.changeset/empty-bananas-wait.md

```diff
@@ -0,0 +1,6 @@
+---
+'@expressive-code/plugin-frames': minor
+'@expressive-code/plugin-shiki': minor
```

Comment:
I've got not experience with npm publishing, so possibly a dumb question:

Do we also need to bump dependent packages like `expressive-code` + remark + astro to make the feature visible to others? If so is that something you'll do later as part of release or should this changeset include them all?

Reply:
I have configured changesets to use the same version number for all packages in the monorepo due to some issues with semver ranges I encountered earlier (which is kind of a brute force fix, but it works). Therefore, the only effect that the package selection here has is which packages will receive the detailed message in their CHANGELOG.md file. The other packages' changelogs will just show that the dependencies were updated.

In cases like this where we add public-facing functionality like a new `ansi` language, I'd probably add the high-level packages to the changeset as well to make the new features visible in their changelog. I will take care of this upon release!